app/models/evaluation_form.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class EvaluationForm < ApplicationRecord
+  belongs_to :challenge
+
+  enum :status, { draft: 0, ready: 1, published: 2 }
+
+  validates :title, presence: true
+  validates :instructions, presence: true
+  validates :challenge_phase, presence: true
```

Comment:
we can leave out the validations for now just to move this forward. based on the latest conversation with Renata,  we will eventually run some validations only when the uses presses save, so it can't be tied to the Rails validation callbacks.